openy_af4_vue_app/package.json

```diff
@@ -12,6 +12,7 @@
     "bootstrap": "^4.6.0",
     "bootstrap-vue": "^2.21.2",
     "core-js": "^3.18.3",
+    "mobile-device-detect": "^0.4.3",
```

Comment:
Are we sure we want to have this library https://github.com/duskload/mobile-device-detect/pulls
A lot of opened security issues @hamrant

Reply:
I'd rather fork it or get maintainer access and fix those security issues

Reply:
@podarok nice catch!

Ok, can try a solution without an additional library with data from `Window.navigator` - https://developer.mozilla.org/en-US/docs/Web/API/Window/navigator (For now it has good browser compatibility).

How to detect iOs device - https://racase.com.np/javascript-how-to-detect-if-device-is-ios/
How to detect mobile device - https://stackoverflow.com/questions/11381673/detecting-a-mobile-browser

```js
const isMobile = navigator.userAgentData.mobile; //resolves true/false
```

cc @manachynskyi

Reply:
Great, let's use it instead of insecure dependency
Thank you @hamrant